jechova.py

```diff
@@ -41,9 +49,20 @@
 
     channel = '#' + args.channel.lstrip('#')
     client = WebClient(token=os.environ['SLACK_API_TOKEN'])
-    client.chat_postMessage(channel=channel, text=dedent('''
-        Máte téma? A mohla bych ho vidět? :eyes:\n\n
-        Pokud už tušíte, o čem Pyvo bude, přidejte to na pyvo.cz — může
-        to udělat kdokoliv, v 5 jednoduchých krocích:
-        https://docs.pyvec.org/guides/meetup.html#informace-na-pyvo-cz
-    ''').strip())
+
+    docs_url = "https://docs.pyvec.org/guides/meetup.html#informace-na-pyvo-cz"
+    if web_url:
+        client.chat_postMessage(channel=channel, text=dedent(f'''
+            Máte téma? A mohla bych ho vidět? :eyes:\n\n
+            Pokud už tušíte, o čem Pyvo bude, přidejte to na pyvo.cz — nejlépe
+            přes tlačítko _přidat informace o dalším srazu_ na <{web_url}>.
+
+            (Existuje i <{docs_url}|podrobnější návod>.)
```

Comment:
Pokud existuje jednodušší návod, nechal bych jen ten. Sekci na docs.pyvec.org bych zachoval pro úplnost, ale tady bych ji už klidně nelinkoval, pokud existuje nějaké nové tlačítko, které to všechno zvládne.